src/pages/using-spark/guides/writing-meaningful-web-content-with-semantic-html.mdx

```diff
@@ -0,0 +1,250 @@
+---
+title: Writing Meaningful Web Content With Semantic HTML
+---
+
+import { SprkHeading } from '@sparkdesignsystem/spark-react';
+
+<span
+  id="writing-meaningful-web-content-with-semantic-html"
+  class="docs-b-Link--anchor"
+></span>
+<SprkHeading
+  element="h1"
+  additionalClasses="sprk-u-mbl sprk-u-Measure"
+  isPageTitle
+>
+  Writing Meaningful Web Content with Semantic HTML
+</SprkHeading>
+
+Typically, in a word document, writers bold or italicize text to add emphasis to the copy.
+
+When producing web content, **HTML** (Hyper Text Markup Language) **elements** define the structure, purpose and intended meaning of a page's web content.
+
+<SprkHeading
+  element="h3"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayThree"
+>
+  Main Takeaways
+</SprkHeading>
+
+- Use HTML elements to define the structure and meaning of a web page's content.
+- Know the purpose of each element to understand when to use it and to ensure the content can be access by assistive technologies.
+- Use the correct element to ensure the content communicates the intended meaning.
+
+Here are some benefits of using semantic HTML
+
+- **It's easier to develop**. It gives content structure making it easier to understand.
+- **It's better for mobile experiences**. Semantic HTML is lighter in file size, and easier to make responsive.
+- **It's good for SEO**. It makes documents more findable, search engines give more importance to keywords inside the content structure.
+
+<span id="table-of-contents" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h2"
+  additionalClasses="sprk-u-mbl sprk-u-Measure"
+  variant="displayTwo"
+>
+  Table of Contents
+</SprkHeading>
+
+- [HTML Elements](#html-elements)
+  - [Creating Elements Using Tags](#creating-elements-using-tags)
+  - [Common HTML Elements For UX writers](#common-html-elements-for-ux-writers)
+- [Semantic Elements](#semantic-elements)
+  - [The Strong Element](#the-strong-element)
+    - [The Strong vs The Bold Element vs Styling Text](#the-strong-vs-the-bold-element-vs-styling-text)
+  - [The Emphasis Element](#the-emphasis-element)
+    - [The Emphasis vs Italic or Idiomatic Text Element](#the-emphasis-vs-the-idiomatic-text-element)
+- [Accessibility](#accessibility)
+
+<span id="html-elements" class="docs-b-Link--anchor"></span>
+<SprkHeading
+  element="h2"
+  additionalClasses="sprk-u-mbm sprk-u-Measure"
+  variant="displayTwo"
+>
+  HTML Elements
+</SprkHeading>
+
+HTML (Hyper Text Markup Language) consists of series of elements, which enclose, or wrap pieces of content they appear or act a certain way.
+
+Elements describe or represent the meaning, structure, or style of a word, to:
+
+- Web browsers
+- Developers
+- Users of assistive technologies
```

Comment:
I love lists, but I don't think this needs to be one.

`Elements communicate the meaning, structure, or context to web browsers, developers and users of assistive technologies.`